modules/update-best-practices.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-best-practices_{context}"]
+= Best practices for cluster updates
+
+{product-title} is designed to provide a robust update experience that allows clusters to update with minimal disruptions to workloads.
+Updates will not begin unless the cluster is determined to be in an upgradeable state at the time of the update request.
+
+This design ensures that some key conditions are met before initiating an update, but there are a number of actions you can take to increase your chances of a successful cluster update.
+
+[discrete]
+[id="recommended-versions_{context}"]
+=== Choose versions recommended by the OpenShift Update Service
```

Comment:
I think it is better to let customers know how to get the OSUS recommended versions, like on web console or command line.

Reply:
Unless I'm mistaken - I think as long as users follow our update procedures then they'll always use recommended versions, unless users look up how to find conditional updates or use `--to-image`. But those two options take more work to find in the docs.

So I think we already show users how to get recommended versions because that's the default method that we document in almost all of our procedures.

Reply:
um...you are right.